### PR TITLE
Add captcha check to faucent fund tokens - Closes #6228

### LINKED ISSUE #6228: Implement captcha for faucet plugin
### Description

Faucet fund requests should check captcha to prevent abuse

Related: https://developers.google.com/recaptcha/docs/v3

### Motivation

Prevent spamming of faucet requests

### Acceptance Criteria

- Implement captcha functionality
- Test

### LINKED ISSUE #6247: Add functional tests for faucet plugin using testing utils
### Description

Faucet plugin requires testing which use the new framework testing utils.

### Motivation

Framework testing utils should be used to write tests once they are merged into development branch to write tests.

### Acceptance Criteria

Write tests using new utils

### PR DESCRIPTION
**Pull request by mitsuaki-u**
### What was the problem?

This PR resolves #6228 

### How was it solved?

Add request to google captcha api from `fundTokens` actions

### How was it tested?

Functional tests to be added once application testing framework is merged into development.

### CODE REVIEW

**Review by ishantiw — CHANGES_REQUESTED** [review]
This should be modified I think `captcha: this._options.captchaSecret`
https://github.com/LiskHQ/lisk-sdk/blob/0db55d4cb9e3f308deeac52bf0a7aa2ff65367cc/framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts#L197

**Comment by ishantiw** [comment]
Also, does this part in UI needs some change or it will be covered in another issue?
https://github.com/LiskHQ/lisk-sdk/blob/0db55d4cb9e3f308deeac52bf0a7aa2ff65367cc/framework-plugins/lisk-framework-faucet-plugin/src/ui/app.tsx#L38-L68

File: framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts [hunk 1]
```diff
@@ -175,9 +193,9 @@ export class FaucetPlugin extends BasePlugin {
 
 		const config = {
 			applicationUrl: this._options.applicationUrl,
-			amount: (BigInt(this._options.amount) / BigInt(10 ** 8)).toString(),
+			amount: this._options.amount,
 			tokenPrefix: this._options.tokenPrefix,
-			captcha: this._options.captcha,
+			captcha: this._options.captchaSecret,
```
Comment (shuse2) [inline framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts:198]: this should be sitekey to send it to the UI. secret only should be used on the server side

File: framework-plugins/lisk-framework-faucet-plugin/src/plugin/defaults/config.ts [hunk 2]
```diff
@@ -43,9 +43,9 @@ export const config = {
 			format: 'uri',
 			description: 'The URL of the logo used on the UI',
 		},
-		captcha: {
-			type: 'object',
-			description: 'The re-captcha config',
+		captchaSecret: {
+			type: 'string',
+			description: 'The re-captcha secret key',
```
Comment (shuse2) [inline framework-plugins/lisk-framework-faucet-plugin/src/plugin/defaults/config.ts:48]: in addition, it requires captcha sitekey here

**Review by shuse2 — CHANGES_REQUESTED** [review]

**Review by shuse2 — APPROVED** [review]

**Review by ishantiw — APPROVED** [review]

**Pull request merged**